xbmc/music/tags/MusicInfoTag.h

```diff
@@ -258,6 +260,7 @@ class CMusicInfoTag final : public IArchivable, public ISerializable, public ISo
   std::string m_stationName;
   std::string m_stationArt; // Used to fetch thumb URL for Shoutcasts
   std::string m_songVideoURL; // link to a video for a song
+  std::map<int, std::vector<std::string>> m_chapters; // Ch No., name, start time, end time
```

Comment:
Not sure I understand what this map actually holds, after reading the comment.

`first`being the ch. number, okay.
`second` being a string vector, holding start AND end time? How does that work? Is it a vector with always two elements, first being start time, second being end time? In this case a would not use a vector, but a std::pair, or even better a dedicated struct with two fields.

EDIT: after reading the code, the vector always seems to hold three elements, so I would suggest to introduce a small struct with three fields, chapterNumber, startTime , endTime. Them its should be quite clear what the map contains and the code readability will increase overall.

Reply:
OK, I've re-written this to use a struct that belongs to CSong  The struct contains the chapter name, the start time and the end time.  The map therefore contains the chapter number as a key and the associated name, start time & end time for that key.

Reply:
Much better now, imo. Thanks.